apps/dapp-console/app/faucet/page.tsx

```diff
@@ -50,8 +54,20 @@ export default function Faucet() {
           />
         </CardHeader>
         <div className="w-full border-t-1 border-border pb-6" />
-        <CardContent>content here</CardContent>
+        <CardContent>
+          <FaucetContent />
+        </CardContent>
       </Card>
+
+      <Tile
+        className="w-full max-w-screen-lg my-10"
+        variant="secondary"
+        title="Apply for bulk tokens →"
+        description="Superchain faucet only dispenses a small number of test ETH each day. Apply to get test tokens in bulk."
+        onClick={() => {}}
+      />
```

Comment:
This should be removed, we no longer provide a bulk testnet form

Reply:
dope, im removing this in the PR at the top of the stack to avoid merge conflicts